x86/src/ops/math.rs

```diff
@@ -808,32 +808,44 @@ pub fn imul_rm8(cpu: &mut CPU, mem: Mem, instr: &Instruction) {
     cpu.regs.set16(Register::AX, res);
 }
 
-fn imul_trunc(x: i32, y: i32, _flags: &mut Flags) -> i32 {
+fn imul_trunc_32(x: i32, y: i32, _flags: &mut Flags) -> i32 {
```

Comment:
Take a look at functions like this, which are generic over the types of integers:

https://github.com/evmar/retrowin32/blob/b998e571ba9e31b4cc95e18f091376cb8372a9fb/x86/src/ops/math.rs#L51

In particular that will let the 16-bit and 32-bit versions will share the (TODO) flag-setting code.

...Also, if your program is calling imul, it might need those flags set, so I filed https://github.com/evmar/retrowin32/issues/57 with some notes.